### PR TITLE
Initial ui work

### PR DESCRIPTION
**Pull request by BrendanJohn**
This pr sets up our ui for a test driven development approach. We can now test our javascript modules using the command npm test. Also included some updates to the readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,5 +128,16 @@ The below checklist should be complete to ensure a story is ready.
 - [ ] The user story is in line with the Product Vision
 - [ ] All blockers and dependencies for the needed implementation are resolved
 
+# Sprint 1 Burndown Chart
+https://docs.google.com/spreadsheets/d/1wAimr26wow8j6nflQbo_E_7oXwHJKrT59--CRPgGTjE/edit#gid=1432140175
+
+# Sprint 1 Evidence of Daily Scrum
+https://github.com/jorgecotillo/green_light_health/blob/master/img/july-2-2020/scrum/sprint1-scrum1.png
```
Comment (noweber) [inline README.md:135]: Can you split this README work into a separate pull request? The evidence of daily scrums, sprint planning, etc is within the 'img' directory at the root of the repository. We should link to that instead. The reason it should be two PRs is because else you are asking for two different types of work to be merged at once, which tends to be problematic with reviewers/comments.

File: src/GreenLightHealth.Client/wwwroot/js/site.js [hunk 2]
```diff
@@ -2,3 +2,22 @@
...
+
+
+if (document.getElementById("accept")) {
+    document.getElementById("accept").addEventListener("click", agreeToHealthDec);
+    document.getElementById("decline").addEventListener("click", declineHealthDec);
+}
+
+//TODO figure out what needs to happen here, maybe just highlight for now
+function agreeToHealthDec() {
+    alert ("@User.Identity.Name agrees to health declaration");
```
Comment (noweber) [inline src/GreenLightHealth.Client/wwwroot/js/site.js:14]: I don't think an alert will work for this. Once dismissed there won't be any persistent visual on screen.

**Review by noweber — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (BrendanJohn) [reply README.md:135]: I'll decline this pr and rework tomorrow
